textkit/transform/tokens_to_tfidf.py

```diff
@@ -0,0 +1,93 @@
+# encoding=utf8
+import click
+import math
+import nltk
+from collections import defaultdict
+from textkit.utils import read_tokens, write_csv
+
+
+# input document_tokens: a list of tokens that represent a document
+# input corpus_tokens: list of lists of tokens.
+#  One list for each document in the corpora.
+# output: list of (token, tf-idf) values
+#         for each unique token in document_tokens
+@click.command('tokens2tfidf')
+@click.argument('tokens', type=click.File('r'), default=click.open_file('-'))
+@click.argument('corpus',  type=click.Path(exists=True), nargs=-1)
```

Comment:
Should we add some text here to indicate that this should be a directory path with all your text files?